aiida_pseudo/groups/mixins/cutoffs.py

```diff
@@ -90,60 +91,92 @@ def validate_stringency(self, stringency: str) -> None:
         if stringency not in self.get_cutoff_stringencies():
             raise ValueError(f'stringency `{stringency}` is not defined for this family.')
 
-    def _get_cutoffs(self) -> dict:
-        """Return the cutoffs dictionary.
+    def _get_cutoffs_dict(self) -> dict:
+        """Return the cutoffs dictionary that maps the stringencies to the recommended cutoffs.
 
         :return: the cutoffs extra or an empty dictionary if it has not yet been set.
         """
         return self.get_extra(self._key_cutoffs, {})
 
+    def _get_cutoffs_unit_dict(self) -> str:
+        """Return the cutoffs units for each of the stringencies.
+
+        :return: the cutoffs units extra or an empty dictionary if it has not yet been set.
+        """
+        return self.get_extra(self._key_cutoffs_unit, {})
+
     def get_default_stringency(self) -> Union[str, None]:
         """Return the default stringency if defined.
 
         :return: the default stringency or ``None`` if no cutoffs have been defined for this family.
-        :raises ValueError: if not default stringency has been defined.
+        :raises ValueError: if default stringency has not been defined.
         """
         try:
             return self.get_extra(self._key_default_stringency)
         except AttributeError as exception:
             raise ValueError('no default stringency has been defined.') from exception
 
+    def set_default_stringency(self, default_stringency: str) -> None:
+        """Set the default stringency for the recommended cutoffs.
+
+        :param default_stringency: the default stringency to be used when ``get_recommended_cutoffs`` is called. If is
+            possible to not specify this if and only if the cutoffs only contain a single stringency set. That one will
+            then automatically be set as default.
```

Comment:
I think these last two sentences of the docstring actually apply to `set_cutoffs` doesn't it? Here it doesn't make much sense.

Reply:
Oops! Copy 🍝 ! 😬